build/wdqs-frontend/README.md

```diff
@@ -2,6 +2,17 @@
 
 Frontend for the [Wikidata Query Service (WDQS)](https://www.mediawiki.org/wiki/Wikidata_Query_Service).
 
+To interact with the WDQS frontend, navigate to the URL defined as `WDQS_FRONTEND_PUBLIC_HOST` in the `.env` file. By default, this is set to `wdqs-frontend.example.com`.
+
+Alternatively, send `GET` requests with your SPARQL query to the WDQS frontend endpoint:
+`https://wdqs-frontend.example.com/proxy/wdqs/bigdata/namespace/wdq/sparql?query={SPARQL}`
```

Comment:
I think this information I would actually expect in `build/wdqs/README.md`, because `wdqs` is the one  responding to those requests and offering this http endpoint on `http://wdqs:9999/bigdata/namespace/wdq/sparql`. 

But the thing you wrote is correct in the `deploy` context. For a `deploy` user wanting to interact with `wdqs` via API from the outside, it is accessible via `https://wdqs-frontend.example.com/proxy/wdqs/bigdata/namespace/wdq/sparql`. Should we mention it in `deploy` and `wdqs` readme? WDYT?